pyuvdata/uvdata/uvdata.py

```diff
@@ -3232,6 +3458,14 @@ def __add__(
                     rtol=this._lst_array.tols[0],
                     atol=this._lst_array.tols[1],
                 )
+            elif a == "_channel_width" and this.future_array_shapes:
```

Comment:
Another case where some added handling for `flex_spw=True` and `this.future_array_shapes=False` would be good (gah, seems like I missed a number of these...).

Reply:
yes, I found this one too.